frappe/geo/country_info.json

```diff
@@ -810,10 +810,10 @@
  },
  "Estonia": {
   "code": "ee",
```

Comment:
Hey, thanks for your first contribution!

Shouldn't the `code` also be updated to `eur` ?

Reply:
I don't think so. It looks like the code is the country ISO code. Take a look for example Greece:

```
"Greece": {
  "code": "gr",
  "currency": "EUR",
  "currency_fraction": "Cent",
  "currency_fraction_units": 100,
  "smallest_currency_fraction_value": 0.01,
  "currency_symbol": "\u20ac",
  "number_format": "#,###.##",
  "timezones": [
   "Europe/Athens"
```